cmd/frontend/graphqlbackend/users_create.go

```diff
@@ -60,6 +62,14 @@ func (r *createUserResult) ResetPasswordURL(ctx context.Context) (*string, error
 	if !userpasswd.ResetPasswordEnabled() {
 		return nil, nil
 	}
+	var ru string
+
+	if conf.CanSendEmail() {
+		ru, err := userpasswd.HandleSetPasswordEmail(ctx, r.user.ID)
+		if err == nil {
+			return &ru, nil
+		}
```

Comment:
What's the rationale to swallow error here?
```suggestion
		if err != nil {
			return nil, err
		}
		return &ru, nil
```

Reply:
If we fail to send the email, there might be an issue with the email configuration. I don't want a failed email config to completely block the site admin from creating users, so I want to fall back to just creating a password reset url and showing it to the site admin.

Reply:
Gotcha! In that case, we should at least log the error with [`log15.Error`](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+log15.error&patternType=literal).

Reply:
Done!